src/main/scala/io/moia/scalaHttpClient/HttpClient.scala

```diff
@@ -123,7 +123,8 @@ abstract class HttpLayer[LoggingContext](
     case response @ HttpResponse(StatusCodes.Success(_), _, _, _)                => Future.successful(HttpClientSuccess(response))
     case response @ HttpResponse(StatusCodes.BadRequest, _, HttpEntity.Empty, _) => Future.successful(HttpClientError(response))
     case response @ HttpResponse(StatusCodes.BadRequest, _, _, _)                => Future.successful(DomainError(response))
```

Comment:
Just saw this, while that's true for our case it's weird in a general library...